expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsViewGestures.kt

```diff
@@ -0,0 +1,32 @@
+package expo.modules.maps
+
+import com.google.android.gms.maps.GoogleMap
+
+class GoogleMapsViewGestures() : MapsViewGestures {
+
+  private var googleMap: GoogleMap? = null
+
+  fun initMap(googleMap: GoogleMap) {
+    this.googleMap = googleMap
+  }
```

Comment:
I think we named markers class `GoogleMapsMarkers` so we can try to keep it same way, although this is very subjective and isn’t issue at all. What do you prefer to do?

We can also try to initialize `googleMap` object using constructor as I proposed above.

```suggestion
class GoogleMapsGestures(map: GoogleMap) : Gestures {

  private var googleMap: GoogleMap = map
```